tests/sync/file.cpp

```diff
@@ -148,15 +148,15 @@ TEST_CASE("sync_file: URL manipulation APIs", "[sync]") {
 TEST_CASE("sync_file: SyncFileManager APIs", "[sync]") {
     const std::string identity = "abcdefghi";
     const std::string local_identity = "123456789";
-    const std::string manager_path = base_path + "syncmanager/";
     const std::string app_id = "test_app_id*$#@!%1";
     const std::string expected_clean_app_id = "test_app_id%2A%24%23%40%21%251";
+    const std::string manager_path = base_path + "syncmanager/mongodb-realm/" + expected_clean_app_id + "/";
     prepare_sync_manager_test();
     auto cleanup = util::make_scope_exit([=]() noexcept { util::try_remove_dir_recursive(base_path); });
-    auto manager = SyncFileManager(manager_path, app_id);
+    auto manager = SyncFileManager(base_path + "syncmanager/", app_id);
```

Comment:
Minor, but not 100% sure why we need the `syncmanager` subfolder here?

Reply:
I mainly converted this test to make sure it worked as it did previously– my guess is that this is something legacy-ish.